scripts/rollup/results.json

```diff
@@ -1,17 +1,17 @@
 {
-  "branch": "master",
+  "branch": "warnForKeysInFragmentsRefactor",
```

Comment:
That's what we do for now. It doesn't really matter and gives us heads up on unexpected changes. Ideally later we'd migrate to another approach (e.g. a bot that comments). But this works okay for now.